httpdocs/js/webots-cloud.js

```diff
@@ -371,15 +396,32 @@ document.addEventListener('DOMContentLoaded', function() {
           const old = document.querySelector('#sync-' + id).parentNode.parentNode;
           const parent = old.parentNode;
           if (data.error) {
-            ModalDialog.run('Project creation error', data.error);
-            parent.removeChild(old);
+            let errorMsg = data.error;
+            if (errorMsg.startsWith('YAML file error:')) {
+              errorMsg = errorMsg + 
+                `<div class="help">More information at: 
+                  <a target="_blank" href="https://github.com/cyberbotics/webots-cloud/blob/beta/README.md#webotsyaml">
+                    github.com/cyberbotics/webots-cloud/webotsyaml
```

Comment:
That link seems misleading, isn't it?

Reply:
I have changed it to matching links (they do not exist yet, but will exist once this PR is merged)